react/src/components/footer/SprkFooter.js

```diff
@@ -103,8 +118,14 @@ class SprkFooter extends Component {
                           return (
                             <li key={id} className="sprk-o-Stack__item">
                               <TagName
-                                className="sprk-c-Footer__link sprk-b-Link sprk-b-Link--light"
-                                href={TagName === 'a' ? href || '#nogo' : undefined}
+                                className="
+                                  sprk-c-Footer__link
+                                  sprk-b-Link
+                                  sprk-b-Link--light
```

Comment:
Is this the right name for this class? I see `sprk-b-Link--inline-light` below in the others but this is set as ` sprk-b-Link--light`